### DISCUSSION TITLE
Fan holder as my first PolyRound application

### DISCUSSION THREAD

**Discussion by fex42** [Irev-Dev/Round-Anything#20] [Show and tell]
Here is a small fan holder that I designed in OpenSCAD using PolyRound. I used it as an opportunity to learn a bit about PolyRound. Using PolyRound helped me a lot to make it not look like my usual square-angled designs, but I guess I only scratched the surface and there is a lot of room for improvements.

<img src="https://i.imgur.com/d45Ejmw.png" width="100%" align="left"> 

You can find the project (including OpenSCAD files) here on Thingiverse:

[https://www.thingiverse.com/thing:4627013](https://www.thingiverse.com/thing:4627013)

There's also an improved version using a larger fan (and a "spoiler" to get some airflow below the board):

<img src="https://i.imgur.com/xQNSLe1.png" width="100%" align="left"> 

[https://www.thingiverse.com/thing:4637905](https://www.thingiverse.com/thing:4637905)

**Comment by Irev-Dev** [comment]
Awesome!

Thanks for sharing @fex42.

**Comment by fex42** [comment]
It looks like I have specialised in creating board fan mounts for 3D printers with PolyRound.
Thanks in no small part to PolyRound, I have now created a variant for the Anycubic i3 Mega that also works very well!

<img src="https://i.imgur.com/BHush9u.png" width="100%" align="left"> 
<img src="https://i.imgur.com/NFLroTJ.png" width="100%" align="left"> 

You can find the project (including OpenSCAD files) here on Thingiverse:

[https://www.thingiverse.com/thing:4749832](https://www.thingiverse.com/thing:4749832)
